src/components/Slider/Slider.js

```diff
@@ -0,0 +1,21 @@
+import "./Slider.css";
+
+const Slider = ({ color, setColor, value }) => {
+  return (
+    <div className="slider">
+      <input
+        id="slider"
```

Comment:
s ID v reactu opatrně - je to unikátní identifikátor na celé stránce, tj tady to znamená že kdybys to ID někde k něčemu potřebovala (a tady není potřeba) tak tuhle komponentu se stejnou základní barvou můžeš použít jenom jednou v celé aplikaci!

Reálně by se mohlo stát že to ID budeš potřebovat kvůli například labelu kde atribut for= prostě bere ID https://www.w3schools.com/tags/att_label_for.asp

V takovým případě se dělá to, že na mount komponenty vygeneruješ náhodné id (např pomocí https://www.npmjs.com/package/uuid)

```
const ColorSlider = ({ baseColor, colorName, onValueChange }) => {
  const [id] = useState(uuid());
...

  return (
    <>
      <label for={id}>{colorName}</label>
      <input
        type='range'
        className={`slider slider--${baseColor}`}
        id={id}
```